packages/components/RadioGroup/src/RadioButton.macos.tsx

```diff
@@ -6,26 +6,41 @@ import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
 import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
 import { RadioGroupContext } from './RadioGroup';
 import { radioButtonName, IRadioButtonProps, IRadioButtonSlotProps, IRadioButtonType } from './RadioButton.types';
+import { useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
 
 const NativeRadioButtonView = ensureNativeComponent('FRNRadioButtonView');
 
 export const RadioButton = compose<IRadioButtonType>({
   displayName: radioButtonName,
   usePrepareProps: (userProps: IRadioButtonProps, useStyling: IUseComposeStyling<IRadioButtonType>) => {
-    const { content, buttonKey, disabled } = userProps;
+    const defaultComponentRef = React.useRef(null);
+    const { content, buttonKey, disabled, componentRef = defaultComponentRef } = userProps;
     const info = React.useContext(RadioGroupContext);
+    const buttonRef = useViewCommandFocus(componentRef);
```

Comment:
@chiuam were talking offline about this and we're pretty sure this makes `buttonRef` null, even with my changes :/ You could try just doing `const buttonRef = componentRef` for the macOS version.

Reply:
Nah it totally works, the debugger shows a huge blob of JSON

Reply:
nm I take it back, componentRef was the thing I had been logging.  buttonRef does seem to be empty.

Reply:
Oh. I continue to be confused by this hook.